web-search-engines/langchain4j-web-search-engine-searchapi/src/test/java/dev/langchain4j/web/search/searchapi/SearchApiWebSearchEngineIT.java

```diff
@@ -0,0 +1,93 @@
+package dev.langchain4j.web.search.searchapi;
+
+import static dev.langchain4j.web.search.searchapi.SearchApiWebSearchEngine.DEFAULT_ENGINE;
+import static dev.langchain4j.web.search.searchapi.SearchApiWebSearchEngine.DEFAULT_ENV_VAR;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+
+import dev.langchain4j.web.search.WebSearchEngine;
+import dev.langchain4j.web.search.WebSearchEngineIT;
+import dev.langchain4j.web.search.WebSearchInformationResult;
+import dev.langchain4j.web.search.WebSearchOrganicResult;
+import dev.langchain4j.web.search.WebSearchResults;
+
+@EnabledIfEnvironmentVariable(named = "SEARCHAPI_API_KEY", matches = ".+")
+class SearchApiWebSearchEngineIT extends WebSearchEngineIT {
+
+	
+	private static final boolean logRequests = false;
+	
+    protected WebSearchEngine webSearchEngine = SearchApiWebSearchEngine.withApiKey(System.getenv(DEFAULT_ENV_VAR)).logRequests(logRequests).build();
+    
+    @Test
+    void search_for_chatgpt() {
+        // given
+        SearchApiWebSearchEngine searchapiWebSearchEngine = SearchApiWebSearchEngine.builder()
+                .apiKey(System.getenv(DEFAULT_ENV_VAR))
+                .logRequests(logRequests)
+                .build();
+
+        // when
+        WebSearchResults webSearchResults = searchapiWebSearchEngine.search("chatgpt");
+
+        // then
+        assertThat(webSearchResults.searchInformation().totalResults() > 0);
```

Comment:
assertThat is not followed by an assertion invocation, Sonar java:S2970. Expected is such assert notation:
```suggestion
        assertThat(webSearchResults.searchInformation().totalResults()).isPositive();
```
the same for the following asserts

Reply:
FYI you could see these code smell warnings from SonarLint plugin

Reply:
Thanks for flagging! The assert code smells should be fixed in `SearchApiWebSearchEngineIT.java` now.

And thanks for the tip to install the SonarLint plugin 👍.